### PR TITLE
(DOCSP-25297): Got rid of return on CLI command in SwiftUI tutorial

### PR DESCRIPTION
**Pull request by lindseymoore**
### Jira

- https://jira.mongodb.org/browse/DOCSP-25297

### Staged Changes (Requires MongoDB Corp SSO)

- [SwiftUI iOS Tutorial](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-25297-cli-typo/tutorial/swiftui/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/tutorial/swiftui.txt [hunk 1]
```diff
@@ -83,8 +83,8 @@ the command below creates an app:
 
 .. code-block:: shell
 
-   realm-cli apps create -n MyTutorialApp --template swiftui.todo.flex 
-      --deployment-model global --location us-va --environment development
+   realm-cli apps create -n MyTutorialApp --template swiftui.todo.flex --deployment-model global --location us-va --environment development
```
Comment (dacharyc) [inline source/tutorial/swiftui.txt:86]: This seems like a reasonable fix based on the Jira request, but this is on two lines in the docs so it's easier to parse visually and doesn't require scrolling within the code example box. I think a better solution would be to use a backslash to escape the return character similar to how you see it in the React Native tutorials: https://github.com/mongodb/docs-realm/blob/f085e97c72ab80d6a4840656eaf86a9f357ae27a/source/tutorial/react-native.txt#L77

```suggestion
   realm-cli apps create -n MyTutorialApp --template swiftui.todo.flex \
      --deployment-model global --location us-va --environment development
```

I think this issue may also be present in the other tutorials. Might be good to check them all and see if you have the reported issue if you use the "copy" icon to copy and paste the command into your terminal and run it.

**Review by dacharyc — COMMENTED** [review]
Back to you!

File: source/tutorial/swiftui.txt (hunk 1 above)
Comment (lindseymoore) [reply source/tutorial/swiftui.txt:86]: Ahh I was trying to figure out how to get the new line without the return, thank you!

**Review by dacharyc — APPROVED** [review]
LGTM! Nice work overcoming git hurdles 👍

**Pull request merged**